cmd/agent/dist/conf.d/jetson.d/conf.yaml.default

```diff
@@ -0,0 +1,2 @@
+instances:
```

Comment:
The name of this file should be `conf.yaml.example`, otherwise it will be enabled by default.